controllers/teams.js

```diff
@@ -1036,6 +1040,10 @@ router.get('/:teamId/members', async (req, res, next) => {
 			} else {
 				actions = addButtonTrash(actions);
 			}
+			let userRole;
+			if (FEATURE_STUDENTS_CANT_BE_TEAM_ADMINISTRATOR_ENABLED) {
+				userRole = roles.find((r) => r._id === user.userId.roles[0]);
```

Comment:
users can have multiple roles, so just comparing with the first role is not ideal, but I suppose it works for students. This could be modeled as "userRolesContainsStudent = user.userId.roles.some(...)"